actions/file_actions.jsx

```diff
@@ -31,7 +31,7 @@ export function handleFileUploadEnd(file, name, channelId, rootId, clientId, {er
             if (res && res.body && res.body.id) {
                 e = res.body;
             } else if (err.status === 0 || !err.status) {
-                e = {message: Utils.localizeMessage('channel_loader.connection_error', 'There appears to be a problem with your internet connection.')};
+                e = {message: Utils.localizeMessage('file_upload.generic_error', 'There was a problem uploading your files.')};
```

Comment:
Is this string still used elsewhere?

Reply:
Good question, looks like it's not. I'll remove it entirely then